Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/wrapper/impl/ConceptEntryWrapperCreator.java

```diff
@@ -24,101 +25,109 @@
  */
 @Component
 public class ConceptEntryWrapperCreator implements IConceptWrapperCreator {
-    
-    @Autowired
-    private IConceptManager conceptManager;
-
-    @Autowired
-    private IConceptTypeManger typesManager;
-
-    @Autowired
-    private IUserManager usersManager;
-    
-    @Autowired
-    private IURIHelper helper;
-    
-    /**
-     * This method creates wrappers for the concept entries passed as parameter
-     * 
-     * @param entries
-     *            Holds the concept entries to be wrapped
-     * @return
-     * @throws LuceneException 
-     */
-    @Override
-    public List<ConceptEntryWrapper> createWrappers(ConceptEntry[] entries) throws LuceneException {
-        List<ConceptEntryWrapper> foundConcepts = new ArrayList<ConceptEntryWrapper>();
-
-        if (entries == null)
-            return foundConcepts;
-
-        for (ConceptEntry entry : entries) {
-            
-            ConceptEntryWrapper wrapper = new ConceptEntryWrapper(entry);
-            wrapper.setUri(helper.getURI(entry));
-            if (entry.getTypeId() != null && !entry.getTypeId().isEmpty())
-                wrapper.setType(typesManager.getType(entry.getTypeId()));
-
-            if (entry.getCreatorId() != null && !entry.getCreatorId().isEmpty())
-                wrapper.setCreator(usersManager.findUser(entry.getCreatorId()));
-
-            // if entry wraps a wordnet concepts, add it here
-            List<ConceptEntry> wordnetEntries = new ArrayList<ConceptEntry>();
-            String wordnetIds = (entry.getWordnetId() != null ? entry
-                    .getWordnetId() : "");
-            {
-                String[] ids = wordnetIds.trim().split(
-                        Constants.CONCEPT_SEPARATOR);
-                if (ids != null) {
-                    for (String id : ids) {
-                        if (id != null && !id.trim().isEmpty()) {
-                            ConceptEntry wordnetEntry = conceptManager
-                                    .getWordnetConceptEntry(id);
-                            if (wordnetEntry != null)
-                                wordnetEntries.add(wordnetEntry);
-                        }
-                    }
-                    wrapper.setWrappedWordnetEntries(wordnetEntries);
 
-                }
-            }
+	@Autowired
+	private IConceptManager conceptManager;
 
```

Comment:
@jdamerow  I am facing the same problem here too. And these  are showing up as changes after rebasing with develop.